cfgov/jinja2/v1/careers/application-process/index.html

```diff
@@ -23,16 +23,31 @@
 {%- endblock %}
 
 {% block hero -%}
-    {{ hero.render({
-        'heading':   'CFPB Hiring Video',
-        'body':      'We are a government agency created after the 2008
-                      financial crisis. We protect American consumers.
-                      Allow us to introduce ourselves.',
-        'image_url': '/static/img/applying-video-screenshot.png'
-    }, {
-        'link_text': 'See the history of the CFPB',
-        'link_url':  '/the-bureau/history/'
-    }) }}
+    {% do global_dict.update(
+            {
+                'hero': {
+                    'heading':   'CFPB Hiring Video',
+                    'body': {
+                        'source': 'We are a government agency created after the 2008
+                                  financial crisis. We protect American consumers.
+                                  Allow us to introduce ourselves.'
+                    },
+                    'image': {
+                                  'upload': {
```

Comment:
Technically this wouldn't be indented this far since it's an object inside `image`, so it's not aligned with body > source.